### PR TITLE
Change the language for content OnboardingTemplate 

### LINKED ISSUE #36: Change the language  for content OnboardingTemplate

### PR DESCRIPTION
**Pull request by thanhdo29**
## Related issues
Resolve #36 
## Summary
Change the languages for content

## How to test
Go to your device's settings to change the language

## Screenshots
Image 1

Image 2

### CODE REVIEW

**Comment by NguyenVanViet2004** [comment]
Create issue for this PR

**Review by truongtdph40209 — DISMISSED** [review]

File: app/index.tsx [hunk 1]
```diff
@@ -31,12 +31,12 @@ export default function Home (): React.ReactElement {
 
   useLayoutEffect(() => {
     if (!isLoading && !firstTime) {
-      router.replace('/(tabs)/home')
+      router.replace('/authentication/Login')
```
Comment (NguyenVanViet2004) [inline app/index.tsx:34]: redirect to home

File: components/templates/OnboardingTemplate.tsx [hunk 2]
```diff
@@ -10,15 +11,89 @@ import getColors from '~/constants/Colors'
 import useDataOnboarding from '~/constants/DataOnboarding'
 import useTranslation from '~/hooks/useTranslation'
 
+const useHandleLogin = (router: Router): VoidFunction => {
+  return () => {
+    router.replace('/authentication/Login')
+  }
+}
+
+const Paginator = (
```
Comment (NguyenVanViet2004) [inline components/templates/OnboardingTemplate.tsx:20]: Can you separate this function into a separate file?

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]

**Review by NguyenVanViet2004 — APPROVED** [review]

**Comment by KMTus** [comment]
image
successful check

**Review by KMTus — APPROVED** [review]

**Pull request merged**